content/numpy/concepts/math-methods/terms/sqrt/sqrt.md

```diff
@@ -0,0 +1,41 @@
+---
+Title: 'Python:NumPy - sqrt'
+Description: 'Use numpy.sqrt() to determine a positive square-root of a number or array'
+Codecademy Hub Page: 'https://www.codecademy.com/catalog/language/python'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+**numpy.sqrt()** is used to determine a positive square-root of a number or array.
+
+The numpy.sqrt() function is useful for mathematical calculations such as the quadratic equation, Pythagorean Theorem, normal distributions, and more.
+
+## How to use numpy.sqrt()
+
+### Getting Started
+
+To get started with NumPy, follow this link [here](https://numpy.org/doc/stable/user/absolute_beginners.html)
+
+[NumPy sqrt](https://numpy.org/doc/stable/reference/generated/numpy.sqrt.html) accepts up to four parameters and returns a non-negative square-root of an array, element-wise.
+
+## Syntax
+
+***x: array_like***
+The values whose square-roots are required.
+
+***out: ndarray, None, or tuple of ndarray and None, optional***
+A location into which the result is stored. If provided, it must have a shape that the inputs broadcast to. If not provided or None, a freshly-allocated array is returned. A tuple (possible only as a keyword argument) must have length equal to the number of outputs.
+
+***where: array_like, optional***
+This condition is broadcast over the input. At locations where the condition is True, the out array will be set to the ufunc result. Elsewhere, the out array will retain its original value. Note that if an uninitialized out array is created via the default out=None, locations within it where the condition is False will remain uninitialized.
+
+*** **kwargs ***
+For other keyword-only arguments, see the [ufunc docs.](https://numpy.org/doc/stable/reference/ufuncs.html#ufuncs-kwargs)
```

Comment:
These are plagiarised from: https://numpy.org/doc/stable/reference/generated/numpy.sqrt.html

Please edit them in your words. Also, use the following format to describe the parameters:
- `array`: describe this parameter
- `out`: describe this parameter
- `where`: describe this parameter